twpca/model.py

```diff
@@ -183,10 +161,26 @@ def __init__(self, data, n_components,
         self._recon_cost = tf.reduce_sum(self._mask * (self._pred - self._data)**2) / self.num_datapoints
         self._objective = self._recon_cost + self._regularization
 
-        # initialize values for tensorflow variables
+        # declare which variables are trainable
+        self._trainable_vars = [v for k, v in self._vars.items() if k != 'warp'] # time, neuron, trial factors
```

Comment:
`warp` shouldn't be in `self._vars`, it should only be in `self._params`. After the refactor, everything in `self._vars` should actually be trainable, yes?